nastaveni/nastaveni.php

```diff
@@ -175,7 +176,8 @@
 //ceny věcí
 $GLOBALS['KURZ_EURO']=KURZ_EURO;
 $GLOBALS['UCET_CZ']=UCET_CZ;
-$GLOBALS['UCET_SK']=UCET_SK;
+$GLOBALS['IBAN']=IBAN;
```

Comment:
Toto není potřeba. Viz rozdíl mezi glob. proměnnou a konstantou, jako glob. proměnná to imo nikde nepoužíváš. Možná by (pokud si to vygrepuješ) šlo smazat i to KURZ_EURO a UCET_CZ jako glob. proměnné, pokud se nikde nepoužívají.

Obecně by taky měly být mezery kolem =, ale protože se to bude celé mazat, je to jedno.